RadClass/RadClass.py

```diff
@@ -95,16 +94,14 @@ def queue_file(self):
         if self.start_time is not None:
             timestamp = self.processor.timestamps[self.processor.timestamps >=
                                                   self.start_time][0]
-            self.start_i = np.where(self.processor.timestamps ==
-                                    timestamp)[0][0]
+            self.start_i = np.abs(self.processor.timestamps - timestamp).argmin()
         self.start_time = self.processor.timestamps[self.start_i]
         self.current_i = self.start_i
 
         if self.stop_time is not None:
             timestamp = self.processor.timestamps[self.processor.timestamps >=
                                                   self.stop_time][0]
-            self.stop_i = np.where(self.processor.timestamps ==
-                                   timestamp)[0][0]
+            self.stop_i = np.abs(self.processor.timestamps - timestamp).argmin()
```

Comment:
Ordering for `start_time` would probably be the same as `start_time`. In this case, we would want where `self.processor.timestamps == timestamp` _or_ the first instead before the stop time, correct? We wouldn't want data being included _after_ the inputted stop time. In that case, this new implementation also works. Except (FYI), this will fail if the inputted `stop_time` is less than all values in the array.

Reply:
So I could see it going either way - there is an argument to be made for ensuring your analysis always includes the start time and stop time.

I think if the documentation is clear, we can do either one, though.